DQMOffline/Trigger/python/METMonitor_cfi.py

```diff
@@ -0,0 +1,32 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.Trigger.metMonitoring_cfi import metMonitoring
```

Comment:
Dear developers,

May I ask where this module is located? I couldn't find it in the expected location:
- **DQMOffline/Trigger/python/metMonitoring_cfi.py**